src/nmr_to_structure/prepare_input/prepare_nmr_rxn_input.py

```diff
@@ -1,92 +1,163 @@
+import logging
+import os
+import random
+import time
+from functools import partial
+from multiprocessing import Manager, Process
 from pathlib import Path
+from typing import List
 
 import click
 import pandas as pd
 from rxn.chemutils.tokenization import tokenize_smiles
+from rxn.utilities.logging import setup_console_and_file_logger
 from tqdm.auto import tqdm
-import random
 
-from .nmr_utils import build_cnmr_string, build_hnmr_string, save_set, split_data
+from nmr_to_structure.prepare_input.nmr_utils import (
+    DEFAULT_SEED,
+    DEFAULT_NON_MATCHING_TOKEN,
+    evaluate_molecule,
+    log_file_name_from_time,
+    make_nmr,
+    save_set,
+    split_data,
+    tanimoto_set_worker,
+)
+
+DEFAULT_TEST_SIZE = 0.1
+DEFAULT_VAL_SIZE = 0.05
+DEFAULT_MAX_SEQ_LEN = 600
+
+random.seed(DEFAULT_SEED)
```

Comment:
This one can have undesired side effect when someone imports this module. Instead, it is better to put this inside the main function.